tableau-databricks/connection-builder.js

```diff
@@ -40,14 +40,27 @@ limitations under the License.
 	// Minimum interval between consecutive polls for query execution status (1ms)
 	params["AsyncExecPollInterval"] = "1";
 
+	// Load ODBC connection string extras
+	var odbcConnectStringExtrasMap = {};
+	const attributeODBCConnectStringExtras = "odbc-connect-string-extras";
```

Comment:
Can you use the constant here?
```suggestion
	const attributeODBCConnectStringExtras = "connectionHelper.attributeODBCConnectStringExtras";
```
Connection helper constants are documented [here](https://tableau.github.io/connector-plugin-sdk/docs/api-reference#connection-helper).

Reply:
I tried using the constant with and without quotes and it doesn't seem to work. My validation is based on logging messages inside the tabprotosrv file. With the explicit string I can see the values of the parameters printed, but not when using the constant. Is this constant going to be available in the 2020.1 release?

Reply:
Apologies for not following up on this. This constant should be available in 2020.1, no quotes. I'm guessing the quotes was causing the failure, it was a typo on my part.

Reply:
@pvanderknyff the issue of odbc-string-extras not workingwas raised to us by several customers. Could we warmfix it to a maintenance release of 2019.3 and 2019.4?
If the new method works in 2020.1 only, how would you suggest we go about backporting it? Should we just create a separate branch for each tableau release, and fix it with verbatim "odbc-connect-string-extras" instead of the constant in the older ones?

For how long the older Tableau releases are supported by Tableau, and what are the severity criteria for an issue to be backported to an older release?

Reply:
First of all, it looks like we may have added this functionality already to Databricks in 2020.1, which just released. We did a bulk update of all the connectors made with the SDK we included in the Tableau installer for that release. Can you double check to see if the scenarios you are looking at work in 2020.1 with the built-in connector?

For the older versions, because of the relatively high cost of backporting to older Tableau releases we generally only backport for high-priority defects (not new features). We don't yet have a process for determining this for partner-built connectors. Thus, I would ask this question of Blair or Sid if the lack of odbc-connect-string-extras not working is blocking enough customers to qualify as something we should put the work into backporting.

Reply:
@pvanderknyff do you mean you modified it on your side?
Could you then contribute your change back to this repo, so that the code stays in sync with what you have? Or would you prefer us to just merge this one, and you'll pick it up?

Reply:
We bulk updated all of our connectors to add odbc connect string extras, the code we added is identical to the code added here. Merging this change and us merging the change we made is functionally identical, so let's merge this one. Apologies for the confusion on this.